fhd_core/gridding/visibility_degrid.pro

```diff
@@ -42,6 +48,24 @@ psf_dim2=2*psf_dim
 group_arr=reform(psf.id[polarization,freq_bin_i,*])
 beam_arr=*psf.beam_ptr
 
+
+if keyword_set(beam_per_baseline) then begin
+    uv_grid_phase_only=1
+    psf_image_dim=(*psf.image_info).psf_image_dim
+    psf_intermediate_res=(Ceil(Sqrt(psf_resolution)/2)*2.)<psf_resolution
+    image_bot=-Floor(psf_dim/2)*psf_intermediate_res+Floor(psf_image_dim/2)
+    image_top=(psf_dim*psf_resolution-1)-Floor(psf_dim/2)*psf_intermediate_res+Floor(psf_image_dim/2)
+
+    l_m_n, obs, psf, l_mode=l_mode, m_mode=m_mode, n_tracked=n_tracked
```

Comment:
Having read through the rest of the code changes, I now understand this line. It would be more clear with `l_m_n` written as a function, I think:
`n_tracked = l_m_n(obs, psf, l_mode=l_mode, m_mode=m_mode)`

Reply:
Done